tests/test_operator.py

```diff
@@ -394,6 +394,33 @@ def test_consistency_perfect_loops(self, tu, tv, ti0, t0, t1):
         pivot = set([j.expr for j in trees[0][-1].nodes])
         assert all(set([j.expr for j in i[-1].nodes]) == pivot for i in trees)
 
+    @pytest.mark.parametrize('exprs', [
+        ('Eq(ti0[x,y,z], ti0[x,y,z] + ti1[x,y,z])', 'Eq(ti1[x,y,z], ti3[x,y,z])',
+         'Eq(ti3[x,y,z], ti1[x,y,z] + 1.)'),
+        ('Eq(ti0[x,y,z], ti0[x,y,z-1] + ti1[x,y,z+1])', 'Eq(ti1[x,y,z], ti3[x,y,z+1])',
+         'Eq(ti3[x,y,z], ti3[x,y,z-1] + ti0[x,y,z])'),
+        ('Eq(ti0[x,y,z+2], ti0[x,y,z-1] + ti1[x,y,z+1])',
+         'Eq(ti1[x,y,z+3], ti3[x,y,z+1])',
+         'Eq(ti3[x,y,z+2], ti0[x,y,z+1]*ti3[x,y,z-1])'),
+        ('Eq(ti0[x,y,z], ti0[x-2,y-1,z-1] + ti1[x+2,y+3,z+1])',
+         'Eq(ti1[x,y,z], ti3[x+1,y-4,z+1])',
+         'Eq(ti3[x,y,z], ti3[x+5,y,z-1] - ti0[x+3,y-2,z+4])')
+    ])
+    def test_consistency_coupled_w_ofs(self, exprs, ti0, ti1, ti3):
```

Comment:
Could we add a docstring to these two tests? They are a bit cryptic without at least a small description, I think.